src/@ionic-native/plugins/advance-file-picker/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+
+export interface IOSFilePickerPosition {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+}
+
+/**
+ * @name Advance File Picker
+ * @description
+ * This plugin does something
+ *
+ * @usage
+ * ```typescript
+ * import { AdvanceFilePicker } from '@ionic-native/advance-file-picker';
```

Comment:
Update the import path.

`import { AdvanceFilePicker } from '@ionic-native/advance-file-picker/ngx';`